### ISSUE TITLE
Stuck at Awaiting Full CMDR Login

### ISSUE THREAD

**Issue by Rowdyhorse4** [EDCD/EDMarketConnector#2296]
**Please complete the following information:**

[//]: # (You can gather most of this information with the EDMC System Profiler)
 
 - Version: 5.12.0+cc6e200
 - Game Version: Odyssey
 - OS: Windows 11/10
 - OS Locale: English_United Kingdom
 - If applicable: Browser Firefox

**Describe the bug**

[//]: # (A clear and concise description of what the bug is.)

Starting EDMC will prop the log in to Frontier Auth page. When i press log in. the browser will prompt opening EDMC which i will press OK. and then nothing happens. EDMC will still be stuck at "awaiting Full CMDR Login".  When repeated using a stock Edge browser. the same thing still happens. 

**To Reproduce**
1. Open EDMC
2. Log in
3. Allow the browser to open EDMC
4. Done

**Expected behavior**

[//]: # (A clear and concise description of what you expected to happen.)

EDMC is supposed to connect with Frontier but it doesnt.

**Screenshots**

[//]: # (If applicable, add screenshots to help explain your problem.)

**Additional context**

<img width="648" alt="Screenshot 2024-09-26 000035" src="https://github.com/user-attachments/assets/fbffe731-80c4-4242-995d-108ddca4a469">

**Please Confirm the Following...**

[//]: # (Add any other context about the problem here.)
- [ Check] I have checked the [Known Issues](https://github.com/EDCD/EDMarketConnector/issues/618) list to ensure this is not a duplicate
- [ Check] I have checked the [Troubleshooting Guide](https://github.com/EDCD/EDMarketConnector/wiki/Troubleshooting) to check for known workarounds

**Logs**
Please attach both the EDMarketConnector.log and EDMarketConnector-debug.log if available.

You can find these logs at `%TEMP%\EDMarketConnector.log` and `%TEMP%\EDMarketConnector\EDMarketConnector-debug.log`

See [Debug Log File](https://github.com/EDCD/EDMarketConnector/wiki/Troubleshooting#debug-log-files) for information on the Debug Log files
[EDMarketConnector.log](https://github.com/user-attachments/files/17135336/EDMarketConnector.log)
[EDMarketConnector-debug.log](https://github.com/user-attachments/files/17135337/EDMarketConnector-debug.log)

**Comment by chennin** [comment]
Please confirm what you mean by "Log in": while EDMC is running, enter Solo/Open/PG such that you are sitting in your ship (or suit).

(Many people open the game but don't enter a game mode, which EDMC needs)

**Comment by Rowdyhorse4** [comment]
Let me try again.  Log in as in Log in from Firefox. When i start up EDMC, it opens a tab on firefox with leads to the Frontier Authentication log in page. Logging in there will not result in any change to EDMC and forcing it (using the local server Bat file)  will result in the page moving to an Error 500

**Comment by Rowdyhorse4** [comment]
Tried it again whilst in game (PG). EDMC is still not responding from "Awaiting full CMDR Login"

**Comment by flaneur93** [comment]
the solution;

Go to installed files,
Start EDMarketConnector - localserver-auth.bat
click update and login

its worked for me

**Comment by Rowdyhorse4** [comment]
> Please confirm what you mean by "Log in": while EDMC is running, enter Solo/Open/PG such that you are sitting in your ship (or suit).
> 
> (Many people open the game but don't enter a game mode, which EDMC needs)

Combining this. With @flaneur93 's solution worked.   

on their own, it errors out. but once in game in PG mode, with the bat file. it worked. 

Thanks everyone!
